package.json

```diff
@@ -9,8 +9,6 @@
   },
   "scripts": {
     "preinstall": "node ./scripts/ensure-yarn.js",
-    "clean:packages": "lerna run --parallel clean",
```

Comment:
Don't think those were that much needed. If any issue arise from lack of such script we can always reimplement them on our own.